server/config.ts

```diff
@@ -15,11 +15,11 @@ function get<T>(name: string, fallback: T, options = { requireInProduction: fals
 const requiredInProduction = { requireInProduction: true }
 
 export class AgentConfig {
-  maxSockets: 100
+  timeout: number
 
-  maxFreeSockets: 10
-
-  freeSocketTimeout: 30000
+  constructor(timeout = 8000) {
```

Comment:
FYI think you can do, and then not define the field:
```constructor(readonly timeout = 8000) {}```

(You can definitely do it with `constructor(private readonly ...){}` - haven't done it with public fields)

Reply:
Agree about the class but didn't want to change too many files at this point